### PR TITLE
Add operator/grafana/dashboard/datasource verification procedure

### PR DESCRIPTION
**Pull request by pleimer**
I am not sure how you differentiate between the command and the output, so I put a '$' before the command in the code blocks.

### CODE REVIEW

**Comment by leifmadsen** [comment]
@pleimer so my general rule has been, if the command is going to result in output being shown, use a `$` at the start of the command, followed by a blank line and then the console output. If the command is alone and is instructive (would be copy/pasted) I've left the leading `$` off to make it easier to copy/paste from the documentation.

File: doc-Service-Telemetry-Framework/modules/proc_setting-up-the-dashboard-to-host-grafana.adoc [hunk 1]
```diff
@@ -39,19 +39,54 @@ oc project service-telemetry
 oc create -f deploy/subscription.yaml
 ----
 
+Verify that the operator launch succeeded:
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_setting-up-the-dashboard-to-host-grafana.adoc:42]:
```suggestion
To verify that the operator launched successfully, run the `oc get csv` command. If the value of the REPLACES column is Succeeded, the operator launched successfully:
```

Bit of a clunky rewrite. In both of your opinions, is it necessary to spell out what a successful launch looks like in the output? Overkill?

File: doc-Service-Telemetry-Framework/modules/proc_setting-up-the-dashboard-to-host-grafana.adoc [hunk 2]
```diff
@@ -39,19 +39,54 @@ oc project service-telemetry
...
+----
+
 . Launch a Grafana instance:
 +
 ----
-oc create -f deploy/grafana.yaml
+$ oc create -f deploy/grafana.yaml
+----
+
+Verify that the grafana instance deployed:
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_setting-up-the-dashboard-to-host-grafana.adoc:58]:
```suggestion
Verify that the Grafana instance deployed:
```

File: doc-Service-Telemetry-Framework/modules/proc_setting-up-the-dashboard-to-host-grafana.adoc [hunk 3]
```diff
@@ -39,19 +39,54 @@ oc project service-telemetry
...
 
+
 . Create the datasource and dashboard resources:
 +
 ----
 oc create -f deploy/datasource.yaml \
     -f deploy/rhos-dashboard.yaml
 ----
 
+Verify the resources installed correctly:
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_setting-up-the-dashboard-to-host-grafana.adoc:75]:
```suggestion
Verify that the resources installed correctly:
```

File: doc-Service-Telemetry-Framework/modules/proc_setting-up-the-dashboard-to-host-grafana.adoc (hunk 1 above)
Comment (leifmadsen) [reply doc-Service-Telemetry-Framework/modules/proc_setting-up-the-dashboard-to-host-grafana.adoc:42]: @JoanneOFlynn2018 actually the column name is `PHASE` (it's a bit clunky as you noted in the output). I think this is fine though. Just change REPLACES to PHASE and we can ship it.

**Comment by leifmadsen** [comment]
Re-upped this after merging the other branch. Let's just do a spot check and make sure everything still looks correct flow-wise.

CC: @pleimer, @JoanneOFlynn2018

**Comment by pleimer** [comment]
Looks good!

**Pull request merged**